src/templates/article.js

```diff
@@ -59,7 +59,7 @@ const ArticlesTemplate = (props) => {
     const side_banner_data_details = {
         max_w_value: '328px',
         max_w_tablet: '320px',
-        isExternal: true,
+        isExternal: footer_banner_data?.cta_url.search(/deriv\.(com|me)/g) === -1 ? true : false,
```

Comment:
AFAIK all side and footer banners are meant to redirect to our sites, so I suppose we don't even need to check the URL, as they might possibly link to future platforms that are not part of this URL.

Reply:
I've found another links that is not to deriv.com...